exchange-contract/README.md

```diff
@@ -1,79 +1,92 @@
 
-## Contibutions of this Repository
+## Create an Issuer for Fungible Assets
 
-We make the following contributions via this repository with regards to the problem statment and solution overview presented above.
+The following examples assume that we want to support the exchange of red marbles and blue
+marbles. We assume the following identities:
 
-### 1. Token-Guardian Protocol:
+* Blue Marbles Player Association (BMPA) -- an oversight organization that tracks blue marble banks
+* Blue Marble Chapter (BMC) -- a local chapter of the BMPA with a large store of blue marbles
+* Red Marbles Player Association (RMPA) -- an oversight organization that tracks red marble banks
+* Red Marble Chapter (RMC) --a local chapter of the RMPA with a large store of red marbles
+* Alice -- owns a number of blue marbles in in BMC
+* Bob -- owners a number of red marbles in RMC
 
-We describe protocols that could be used to implement the layer 2 NFT token object (see picture above) as well its interactions with the guardian service. The protocol is not restricted to a specific use-case, meaning we do not specify the nature of the confidential asset, and the nature of the capability execution engine located within the Guardian Service. This protocol does not make specific assumptions on Layer 1, the L1/L2 bridge and the marketplace. In other words, the protocol focuses on the following sub-system of the picture above, and consists only the Layer 2 and the Guardian service. Note that in the picture below, in addition to the NFT token object smart contract, we also have a Token Issuer Smart Contract which acts as a pont of entry for a prospective end-points (Alice in above picture) that wants to deploy a NFT smart contract. The protocol describes the steps for system initialization, creating (minting) the NFT token object, transfering token ownership and finally, invoking operations on the confidential asset subject to policies specified via the NFT token object. Please see [Token Guardian Protocol](./doc/Token_Guardian_Protocol_Description.pdf) for a more detailed description of the various steps involved in the protocol. 
+### Setup
 
-<p align="center">
-  <img src=./doc/token_guardian_arch.png width="700">
-</p>
+The asset type and vetting organization contract objects allow for the establishment of a trust anchor to verify the integrity of exchange operations. For each of the asset types, we need to set up the asset type, the vetting organization, and one or more issuers. We'll walk through the sequence of steps necessary for blue marbles; the same step is used to establish the trust anchors for the red marbles.
```

Comment:
Nit: what do you mean by "integrity of exchange operations"?

Reply:
Done